src/node_perf.cc

```diff
@@ -30,63 +29,106 @@ uint64_t performance_v8_start;
 uint64_t performance_last_gc_start_mark_ = 0;
 v8::GCType performance_last_gc_type_ = v8::GCType::kGCTypeAll;
 
+// Initialize the performance entry object properties
+inline void InitObject(PerformanceEntry* entry, Local<Object> obj) {
```

Comment:
`const PerformanceEntry& entry`? Makes it a bit more obvious which parameter is the source/target